dask/array/tests/test_array_core.py

```diff
@@ -355,8 +355,8 @@ def test_concatenate():
     pytest.raises(ValueError, lambda: concatenate([a, b, c], axis=2))
 
 
-@pytest.mark.parametrize('dtypes', [(('>f8', '>f8'), '>f8'),
-                                    (('<f4', '<f8'), '=f8')])
+@pytest.mark.parametrize('dtypes', [(('>f8', '>f8'), 'float64'),
+                                    (('<f4', '<f8'), 'float64')])
```

Comment:
Why this change?

Reply:
Because this is the `dtype` that running that operation within NumPy returns. So in this particular test, if you do `np.concatenate(arrs)`, they will both have `dtype=float64`. Now that `_meta` is responsible for carrying the `dtype`, we don't carry the result as it was in this test anymore (`>f8` or `=f8`). Do you think this breaks compatibility with anything else?